src/proxy/http/metrics/report.rs

```diff
@@ -126,3 +142,49 @@ where
         Ok(())
     }
 }
+
+// === impl Scope ===
+
+impl Default for Scope {
+    fn default() -> Self {
+        Self {
+            request_total_key: "request_total".to_owned(),
+            response_total_key: "response_total".to_owned(),
+            response_latency_ms_key: "response_latency_ms".to_owned(),
+        }
+    }
+}
+
+impl Scope {
+    fn prefixed(prefix: &'static str) -> Self {
```

Comment:
Since the prefix is just passed to a `format!` invocation, I don't think it needs to be `'static`. `&str` should be sufficient (or even `T: fmt::Display`).

Reply:
Yeah, good point. Let's change this if we ever want to use anything besides a static str ;)